### <p align="center">Exploring the limits of bacteriophage diversity: an in-depth analysis of <br>the non-contractile tail tips among Proteobacteria-infecting phages</p>
<p align="center">Tatiana Lenskaia<sup>1,*</sup>, Sherwood Casjens<sup>2,3</sup>, and Alan Davidson<sup>1,4</sup></p>

<p align="center"><sup>1</sup> Molecular Genetics, Temerty Faculty of Medicine, University of Toronto, Toronto , Canada</p>
<p align="center"><sup>2</sup> School of Biological Sciences, University of Utah, Salt Lake City, USA</p>
<p align="center"><sup>3</sup> Pathology Department, School of Medicine, University of Utah, Salt Lake City, USA</p>
<p align="center"><sup>4</sup> Department of Biochemistry, University of Toronto, Toronto, Canada</p>
<p align="center"><sup>*</sup> Corresponding author: t.lenskaia@utoronto.ca</p>

#### Abstract
<p align="left">The staggering diversity of bacteriophages seems to exceed human comprehension and defy systematic understanding. In this study, we focus on the non-contractile tail tips of phages that infect Proteobacteria, a group of phages notable for both their ecological significance and their sequence variability. Yet, by focusing on this specific group of phages, we uncover patterns that bring clarity to this apparent complexity. Despite substantial divergence in tail tip protein sequences, our analysis reveals recurring organizational features that simplify their interpretation and highlight underlying functional determinants. This demonstrates that even in highly variable regions of phage genomes, systematic understanding is within reach. We further suggest that this perspective extends beyond tail tips, providing a framework for studying phage proteins at large. By narrowing the scope to carefully defined structural components, the seemingly limitless complexity of bacteriophages becomes progressively more approachable, bringing us closer to understanding of the basis for their diversity and evolutionary success.</p>

#### Keywords
<p align="left">Morphogenetic proteins, comparative genomics, structural prediction</p>

#### 1.	Introduction

<p>Bacteriophages, viruses that infect bacteria, are the most abundant biological entities on Earth and exhibit an extraordinary range of strategies for recognizing their hosts and delivering their genomes into their host cells. At the molecular level, these strategies are encoded in phage genomes. Among these strategies, tailed phages are by far the most prevalent. These phages have tail-like structures that serve as devices for detecting and binding to specific bacterial surface receptors and as conduits for delivering viral DNA into the host cell. Although tailed bacteriophages are highly diverse, they exhibit three main tail morphologies: short-tailed podophages, long contractile-tailed myophages, and long non-contractile-tailed siphophages. Gene analyses of virion assembly proteins reveal that the three phage morphotypes share strong similarities in head and DNA-packaging proteins, but differ markedly in their tail proteins, reflecting their distinct tail structures. The number of complete tailed phage genome sequences in the public database has grown to at least tens of thousands. As of September 9, 2025, Caudoviricetes (complete genomes): NCBI GenBank – 23,363 entries; NCBI RefSeq – 5,145 entries.  A large fraction of their genes has no specific functional prediction due to failure of current annotation pipeline methods: about 50-70% of genes are annotated as hypothetical or putative function.</p>
<p>Our approach to understanding the nature of phage diversity is to examine limited representative panels of phage genome sequences in substantially more detail than is possible using all known phage genomes (Casjens, Sherwood R. & Grose, 2016; Casjens, Sherwood R. et al., 2022; Zheng et al., 2025). The nature of such “slices” of extant diversity thus revealed can then be used to extrapolate insights and guide further examinations of the wider diversity.  In this study we examine tail tip structures of siphophages that infect the Proteobacteria clade of phage hosts.</p>
<p>Over long evolutionary timescales, many phage proteins have diverged beyond recognizable sequence similarity, leaving genome annotation pipelines unable to assign functions. This is especially true for the unusually large number of small proteins encoded by phages. (Fremin et al., 2022) reported over 40,000 small-gene families in ~2.3 million phage genome contigs, with small genes being roughly three-fold more prevalent in phage genomes than in their bacterial hosts.  Many of phage proteins are often labeled as “hypothetical proteins” or given vague designations such as “phage protein.” As a result, a substantial fraction of phage genes appears uncharacterized, creating the illusion of an incomprehensibly vast repertoire of novel genes (Paul et al., 2002). This challenge is compounded by inconsistent nomenclature: proteins with the same function are often described differently across phage systems, obscuring true homology and complicating systematic study. While it is undeniable that phages harbor genuine innovations, we argue that a significant fraction of these seemingly novel proteins is in fact distant homologs of known structural components whose functions can now be recognized as such.</p>
<p>For decades, progress in resolving phage structure was slowed by the limits of crystallography and the difficulty of analyzing large, flexible assemblies. Recent advances in cryo-electron microscopy (cryo-EM) computational structure determination methods have made it possible to analyze intact virions at near-atomic resolution, leading to detailed structural models of some tailed phages. These include a number of siphophages infecting Proteobacteria such as lambda, where (Wang et al., 2024) resolved the distal tail region, (Ge & Wang, 2024) demonstrated conserved and flexible elements across related phages, and (Xiao et al., 2023) linked structural differences to host specificity. (Chen et al., 2025) extended this to T1, while structures of Chi (Sonani et al., 2024), JBD30 (Valentová et al., 2024), R4C (Huang et al., 2023), and gene transfer agent RcGTA (Bárdy et al., 2020) further expanded the spectrum of tail-related assemblies. The T5 system has been dissected in particular depth, beginning with receptor-binding proteins solved by crystallography (Flayhan et al., 2014), followed by cryo-EM analyses of the distal tail complex (Degroux et al., 2023; van den Berg et al., 2022), and culminating in studies linking structural transitions to DNA ejection (Linares et al., 2023). (Ayala et al., 2023) extended these insights to DT57C, highlighting how closely related phages can diversify to infect different E. coli strains.</p>
<p>These structural achievements, while transformative, now serve primarily as anchoring points for computational biology. The arrival of AlphaFold (Jumper et al., 2021) and a myriad of the subsequent structure prediction methods has moved the resource-intensive and laborious task of protein fold determination into the computational realm. What once required years of experimental work can now be explored at scale, with cryo-EM acting as a reference framework for validation rather than a universal requirement. Our in-depth examination of solved phage structures has revealed general organizing principles of infection complexes, enabling us to detect related architectures even in phage genomes lacking structural data. This shift from experimental to computational diagnosis of molecular function mirrors a broader trend familiar from medicine, where increasingly accurate and precise diagnostics have historically laid the foundation for disease classification and culminated in codified systems such as the International Classification of Diseases. In phage research, by contrast, our ability to classify pathogens of bacteria remains comparatively young, but the same lesson applies: robust diagnostic methods - whether structural, molecular, or computational - are indispensable for developing reliable and meaningful phage taxonomy.</p>
<p>Our study builds directly on this foundation. By performing a large-scale computational analysis anchored in the experimentally determined structures, we systematically explored the limits of phage diversity at the structural and functional levels. Through in-depth computational analysis, protein sequence and structure comparisons, and protein structure predictions, we identified major organizational themes in infection complexes and demonstrated that these themes recur across highly divergent phage protein families. This analysis not only increases the fraction of phage proteins with plausible functional annotation but also provides a unifying framework for describing phage mosaic architecture and constraints on gene exchange. Most importantly, it shows that computational molecular biology has matured to the point of defining paradigmatic structural and functional solutions in phages at scale. In doing so, our work pioneers the transition from individual structural case studies to a comprehensive computational exploration of phage diversity, establishing a foundation for systematic discovery of the repeating themes that are exhibited by the most abundant and diverse biological entities on Earth.</p>
<p>In this study, we examine tail tip structures in phages that infect Proteobacteria hosts (since it is not yet widely accepted, we do not use the recently proposed term Pseudomonadota for this Phylum level classification). Our collection of siphophage genomes (~42,000 genes in total) encompasses phages infecting hosts from 40 bacterial genera across three classes - Alpha-, Beta-, and Gammaproteobacteria. We understand that it is not exhaustive in representing of the full diversity of Proteobacteria phages in nature, as sequencing efforts are biased toward phages of bacteria that are pathogenic to humans, livestock, or crops. In addition, many Proteobacteria host genera remain unstudied. Nevertheless, we are confident that the size and diversity of our panel make it a valuable resource for systematic exploration of phage diversity.</p>
<p>First, we demonstrate that these diverse siphophage tail tips can be classified into distinct “tail tip types” that have not been previously delineated and examine these types in detail. Second, we show that, using HMM analysis (Finn et al., 2011), PSI-BLAST comparisons (Altschul et al., 1997), and AlphaFold3 (Abramson et al., 2024) structure predictions, all core proteins of the tail tip components in a large panel of Proteobacterial siphophages can be unambiguously identified, allowing the nature of their diversity to be understood. This approach substantially increases the number of encoded proteins with predicted functions and consequently reduces the number of genes that might otherwise be perceived as potentially novel. Third, we describe the mosaic organization of these tail tip components and the apparent constraints on the free exchange of genes among phages. Finally, we propose a streamlined universal nomenclature for siphophage tail tip proteins, which should facilitate future analyses and make this area of research more accessible to readers beyond the immediate field.</p>

#### 2.	Results

##### 2.1.	 Characterizing conservative tail tip features of long-tailed phages

<p>Long-tailed bacteriophages, including siphophages infecting both monoderm and diderm hosts as well as myophages, share important structural and functional similarities in their tail tips. The primary role of the tail tip complex is to make way for successful genome delivery into the bacterial host cell. To achieve this, the tail tip must perform a series of essential functions - recognizing a susceptible bacterium, adhering to its surface, and penetrating its cell envelope to create a conduit for phage DNA ejection.</p> 
<p>Despite employing distinct infection strategies, myophages and siphophages face similar challenges imposed by the complex organization of bacterial cell envelopes. In diderm bacteria, the envelope consists of an inner cytoplasmic membrane, a thin peptidoglycan layer, and an outer membrane rich in lipopolysaccharides that limits macromolecular entry. By contrast, monoderm bacteria possess a single cytoplasmic membrane surrounded by a thick, highly cross-linked peptidoglycan layer embedded with teichoic acids. These structural differences create unique mechanical and biochemical barriers for phage infection, requiring distinct evolutionary adaptations.</p>
<p>Myophages, which possess contractile tails, penetrate the cell envelope by a forceful mechanical process: contraction of the tail sheath drives the tail tube through the cell envelop layers, enabling direct DNA injection into the cytoplasm. Siphophages, which lack a contractile sheath, employ alternative strategies. Those infecting diderm bacteria exploit existing outer membrane porins such as FhuA or LamB as docking sites and utilize enzymatic tail-associated proteins to locally degrade the peptidoglycan, allowing more passive translocation through the envelope. In contrast, siphophages infecting monoderm bacteria rely on enzymatic and structural adaptations tailored to breaching their host thick cell wall. Despite the divergent approaches, both phage types must solve the same fundamental problem - overcoming the multilayered bacterial envelope - to ensure efficient genome delivery.</p>
<p>Across long-tailed phages, there are key functional modules that are consistently identifiable in the tail tip complex: the Distal Tail (DT) and Tail Hub (TH), and Central Fiber (CF)/ Central spike components. These represent the most conserved architectural elements among diverse long-tailed phages. The DT, typically exhibiting a conserved TT-fold, acts as a structural “strap belt” linking the tail shaft and tip components. In addition to this anchoring role, the DT often carries auxiliary domains that function as a “utility belt,” providing docking surfaces for enzymatic or receptor-binding elements. The TH serves as a structural hub, coordinating the transition between the 3-fold symmetry of the tail tip and the 6-fold symmetry of the tail shaft, ensuring mechanical stability and symmetry alignment during infection. The CF elements extend outward from the TH, forming fibers that participate in host recognition and attachment.</p>
<p>In siphophages infecting Gram-positive bacteria, these modules are represented by homologous yet specialized proteins: the DT is often referred to as a Dit-like protein, while the fused TH–CF component corresponds to a Tal-like protein. Figure 1(A) highlights these structural modules in three exemplar phages - a Gram-negative siphophage (lambda), a Gram-positive siphophage (phi80alpha), and a myophage (Mu). Figure 1(B) provides a closer view of the hub junctions (DT–TH interfaces) in these phages, illustrating the remarkable conservation of the tail tip’s core architecture across the long-tailed phage lineages.</p>

<p align="center"><img alt="Figure 1." src="https://github.com/user-attachments/assets/9501f121-e64d-4121-a89b-fd034ff018dc" /></p>

<p align="center"><strong>Fig.1.</strong> Conserved tail tip components in long-tailed phages: <br>
(A) tail tip and (B) tail hub junction.</p>


